src/doom/r_data.c

```diff
@@ -376,7 +376,7 @@ void R_GenerateComposite (int texnum)
 
     // Now that the texture has been built in column cache,
     //  it is purgable from zone memory.
-    Z_ChangeTag (block, PU_CACHE);
```

Comment:
This one is wrong. The memory `block` points to has been `Z_Malloc()`ed, so there is no lump to be released. Please revert back to the original code.